### DISCUSSION TITLE
Move CLOSE WITH COMMENT button to the left

### DISCUSSION THREAD

**Discussion by RobTillaart** [community/community#77835] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

In both issues and pull requests the CLOSE button and the COMMENT button are side by side.
This causes unintended closures and reopening from time to time.

By moving the CLOSE / CLOSE with COMMENT button to the left and create more distance between these buttons this kind of errors will be reduced a lot. Although it might not happen often the result could be that an open issue got lost, which can be pretty serious.

**Comment by dakshithadissanayaka** [comment]
Yes, it's true that the close button and the comment button are very close together in both issues and pull requests, which can lead to accidental clicking and unintended closure or reopening. Placing the close button further away from the comment button would certainly help reduce this risk.

Here's a visual representation of the current layout:

[Image of the current layout of the close button and comment button in GitHub issues and pull requests]
<img width="362" alt="c3Ix5" src="https://github.com/community/community/assets/45692186/0932473d-9823-4e3b-ae8a-62f3dab2081d">


As you can see, the close button is right next to the comment button, making it easy to accidentally click the wrong one. Moving the close button to the left, as suggested, would create more space between the two buttons and make it less likely for users to click the wrong one.

Here's a visual representation of the proposed layout:

[Image of the proposed layout of the close button and comment button in GitHub issues and pull requests]
<img width="686" alt="VFbgo" src="https://github.com/community/community/assets/45692186/911c7c71-8073-46ec-8e83-b6e7d6b24188">


In this layout, the close button is clearly separated from the comment button, making it less likely for users to accidentally click the wrong one. This simple change could help reduce the number of unintended closures and reopenings, ensuring that open issues don't get lost due to accidental clicks.
